packages/fx-core/resource/package.nls.json

```diff
@@ -614,6 +614,17 @@
   "core.collaboration.error.dotEnvFileNotExist": "Failed to find your .env File.",
   "core.collaboration.error.failedToLoadDotEnvFile": "Failed to load your .env File. Reason: %s",
   "core.collaboration.error.failedToGetTeamsAppId": "Failed to find Teams app id in your .env file. Please correctly set the value with key '%s' in your .env file.",
+  "core.collaboration.error.manifestFileNotExist": "Failed to find your manifest file: %s.",
+  "core.collaboration.error.manifestFileNotValid": "Your manifest is invalid: %s. Please make sure 'id' key exists in your manifest file.",
+  "core.collaboration.error.manifestIdNotValid": "Value of 'id' in your manifest is invalid: %s. You can either hardcode your App Id or reference env in the manifest.",
+  "core.collaboration.error.failedToLoadManifest": "Failed to load your manifest file. Reason: %s.",
+  "core.selectAadAppManifestQuestion.title": "Select your AAD manifest.json file",
+  "core.selectTeamsAppManifestQuestion.title": "Select your Teams manifest.json file",
+  "core.selectCollaborationAppTypeQuestion.title": "Select app your want to manager collaborator",
```

Comment:
Select collaborator for your app
or
Select app you want to manage the collaborators

depends on what is really expected the user to select.

Reply:
Choose second one since we are asking user to choose teams app/aad app.